openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -26,6 +26,12 @@ <h1>$_('Community Edit Requests')</h1>
     $desc <a href="$href">$link_text</a>
   </div>
 
+</div>
+$ page = int(input(page=1).page)
```

Comment:
We only need one variable that tracks the page number.  Can you remove the second `page` assignment that is near the end of this file?

Reply:
Alright!